config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -89,6 +89,57 @@ presubmits:
           env:
           - name: WINDOWS_SERVER_VERSION
             value: "windows-2022"
+          - name: GINKGO_FOCUS
+            value: \[sig-windows\] # run just a subset to speed up testing time
     annotations:
       testgrid-dashboards: sig-windows-presubmit
       testgrid-tab-name: pull-e2e-capz-windows-containerd-extension
+  - name: pull-e2e-capz-containerd-windows-2022-extension-gmsa
+    interval: 3h
```

Comment:
is `interval` valid for PR jobs?

Reply:
hmm i copied it from the pull above https://github.com/kubernetes/test-infra/pull/26830/files#diff-88d0d69a1602345889b68d029b226c64238bbb6524ddcf520d6148f828b90105R55

doesn't seem to make sense but doesn't seem to fail

Reply:
removed it